### ISSUE TITLE
Create relationship view

### ISSUE THREAD

**Issue by nightsh** [tenders-exposed/elvis-ember#201]
When clicking on a relationship, one should be a ble to see it in a similar fashion to supplier/procurer pages.

**Comment by nightsh** [comment]
Ping @zufanka @chutko can we have a written record of what it's needed to be shown in the relationship details view?

We talked about it, but I can't find the notes -_-

**Comment by chutko** [comment]
here, i have designed what came to my mind. hope to hear some more ideas from you guys. 

as i see it, a relationship in the network view is a group of contracts (and unsuccessful bids) between specific supplier and procurer over a specific time period. therefore these data seem interesting to me: 

- total money flow
- total contracts
- total bids
- avg competition (=if there is a bid between these two entities, how much competition is there?)
- data missing (total data missing amongst all contracts between these two entities)
- bid success % = (total contracts / total bids) * 100
- red flags in this relationship (we could also list them)
- relationship strength - could be interesting to have a function evaluating the relationship into a value represented by a word: e.g. uninteresting, healthy, suspicious, dangerous
- also a timeline graph of money flow/contracts/relationship strength/ could be useful

![elvis--relationship](https://cloud.githubusercontent.com/assets/4135015/26442665/c1397ff8-4135-11e7-9908-9327c4b0283d.png)

**Comment by zufanka** [comment]
lovely!

ideas:
- total bids: what does this say? Is it total bids of that company on tenders of that government? Because we often do not have information on losing bidders.
Or is it total bids that took place in the bidding for all the contracts these two parties ever did together? Then I would opt for some sort average, such as average amount of bids per tender, but that would already be "average competition"
- bid success % : the same. We more often do not have the information which other companies have also bid on a certain tender. It may be in the Digiwhist data and it may be available on incidental cases, but not from TED at least. Therefore might often be empty or no data. I would leave it for later when we are sure we have that info.
- relationship strength: this indeed could be interesting, but also dangerous :) As we are not sure whether the red flags we see are indeed of corruption or of some hidden market dynamics. I would opt for something like domination: (tenders won by this company / all tenders of this government) *100 - but this is only applicable in the context of the same market and not in for example company centric network (oh my -.-)
- also a timeline graph: yes! We can build on what Ioana already started, right?

**Comment by georgiana-b** [comment]
Sounds like this should contain all edge red flags, including "same city", right?

I would remove the whole row that starts with "Relationship: Healthy"  because we can't make that claim and the other info is already visible in different ways. For example total common contracts is also shown in the "Common Contracts" list.

I agree with Adriana that a timeline graph would be very useful to get insights about how a relationship evolved over time which journalists can correlate with changes in mandates and laws.

**Comment by georgiana-b** [comment]
As I mentioned  in the  backend issue, it's pretty complicated to build this functionality in our current setup  which is one of the reasons to switch to graph dbs.
Please postpone this until we have the new backend.

**Comment by chutko** [comment]
@zufanka can you suggest a list of stats for relationship? what i had designed in there is quite a blind shot :) so what interesting data about the relationship do we have now?

**Comment by chutko** [comment]
@georgiana-b maybe we could mark the issue as blocked

**Comment by georgiana-b** [comment]
Blocked on tenders-exposed/elvis-backend#54

**Comment by nightsh** [comment]
Can we close this for now and open a new issue when tenders-exposed/elvis-backend#54 will be fixed? The new issue will likely be more about completing the view, but now we have the template, it shows some data, and we need to mark its progress somehow :)

**Comment by zufanka** [comment]
@nightsh yes sure ⬆️
